<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditOutFolder">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>721</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditImageFolder">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>721</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>D:\AdditiveManufacturing\set1sample2raw</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenImageFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Image Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenOutFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Out Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenMaskFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Open ROI File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMaskFile">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>721</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>D:\AdditiveManufacturing\ROIS2.tif</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>591</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditRegexImageFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>.+.tif+</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetImageFiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>571</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFileToProcess">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameMode">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>161</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBoxShowInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show input</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelScaleBase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowTiffInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Tiff Info</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowMatInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Mat Info</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAutocleanOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto clean output text</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSaveOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Output</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowMask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show mask</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowMaskAsContour">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show contours only</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxImageScale">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="prefix">
      <string>1 : </string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.250000000000000</double>
     </property>
     <property name="maximum">
      <double>8.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.250000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxImRotate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ImRotate</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMinNorm">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>230</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <double>-65536.000000000000000</double>
     </property>
     <property name="maximum">
      <double>65536.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>256.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxNorm">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>260</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <double>-65536.000000000000000</double>
     </property>
     <property name="maximum">
      <double>65536.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>256.000000000000000</double>
     </property>
     <property name="value">
      <double>65535.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinNorm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Min Norm</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxNorm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Norm</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show out</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxNormalization">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>65</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowCombined">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Combined</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>140</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="spinBoxThreshold">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>40</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxErosionRadius">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>70</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Erosion Radius</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxThresholdImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold Image</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>240</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Diameter</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxOffset">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>62</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxDiameter">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>11</number>
     </property>
     <property name="maximum">
      <number>201</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>91</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCreateTileROIs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Tile ROIs</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>450</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonProcessAllFiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Process all files</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSaveTileRoi">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Tile Roi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonGetImCombination">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Im Combination</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSaveCorrecteImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Correcte Image</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>330</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBoxBackgroundCorrection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Background Correction</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxBlurFilter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Blur Image</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxBlurKernelSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>19</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
